doc/en/components/charts/chart-types-line.md

```diff
@@ -6,55 +6,95 @@ mentionedTypes: ["XamCategoryChart", "XamDataChart"]
 ---
 # $PlatformShort$ Line Chart
 
-<!-- TODO add introduction with info about using category-chart with the chartType property set to Line -->
+A Line Chart, or Line Graph, is a type of category line graph that shows the continuous data values represented by points connected by straight line segments of one or more quantities over a period of time. It’s often used to show trends and perform comparative analysis. The Y-Axis (labels on left side) show a numeric value, while the X-Axis (bottom labels) show a time-series or comparison category. You can include one or more data sets to compare, which would render as multiple lines in the chart. 
+
```

Comment:
I think we should add your first line chart example below here as well so that we have a visual above the fold. 
@tosharky @HUSSAR-mtrela

Reply:
Or maybe it will be better to add the full example with the 3 series so we showcase it early on?